src/Module/Api/Mastodon/Search.php

```diff
@@ -83,18 +83,11 @@ private static function searchAccounts(int $uid, string $q, bool $resolve, int $
 	{
 		$accounts = [];
 
-		if (!$following) {
-			if ((strrpos($q, '@') > 0) && $resolve) {
+		if ($resolve) {
+			if ((strrpos($q, '@') > 0)) {
```

Comment:
Superfluous parentheses.
```suggestion
			if (strrpos($q, '@') > 0) {
```